targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # ==================================================================
 # module list
 # ------------------------------------------------------------------
-# python        3.7       (apt)
+# python        3.8       (apt)
 # jupyter       latest    (pip)
 # jupyterlab    latest    (pip)
-# pytorch       1.6.0     (docker-hub)
+# pytorch       1.7.0     (docker-hub)
 # tensorflow    2.3.0     (pip)
 # ==================================================================
 
-FROM pytorch/pytorch:1.6.0-cuda10.1-cudnn7-devel
+FROM pytorch/pytorch:1.7.0-cuda11.0-cudnn8-devel
```

Comment:
why do we use `devel`? `runtime` is 2G comparing to devel's 5G. See the [difference](https://stackoverflow.com/a/59168833/4977823).

Reply:
As I know, `runtime` image does not provide nvcc. So, you can build `apex`, it's just one lib that requires `nvcc`.